guide/artifact-publishing.md

```diff
@@ -9,10 +9,10 @@ title: "Artifact Publishing"
 
 The `manipulation` plugin tries to detect a publishing plugin used by a project. It currently recognizes two plugins:
 
-* the legacy `maven` plugin,
-* the `maven-publish` plugin.
-* the `gradle-nexus` plugin (https://github.com/bmuschko/gradle-nexus-plugin)
-  * this is a wrapper around the legacy `maven` plugin.
+* the legacy `maven` plugin
+* the `maven-publish` plugin
+* the `gradle-nexus` plugin, [here](https://github.com/bmuschko/gradle-nexus-plugin)
+* this is a wrapper around the legacy `maven` plugin
```

Comment:
I think the original indentation was meant to state `gradle-nexus` is a wrapper around the maven plugin - which I think is true...

Reply:
I didn't release that. It wasn't clear to me at first glance that this had some sort of additional meaning.

I am happy to put it back, or I could just combine it with the previous sentence:

* the `gradle-nexus` plugin, [here](https://github.com/bmuschko/gradle-nexus-plugin), which is a wrapper around the legacy `maven` plugin".

Reply:
Yeah thats fine :-) Thanks!